runtime/jilgen/jilconsts.c

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1991, 2022 IBM Corp. and others
+ * Copyright (c) 1991, 2021 IBM Corp. and others
```

Comment:
This copyright shouldn't be reverted to 2021, the file is modified by other commits in 2022.